docs/rootfs-and-kernel-setup.md

```diff
@@ -46,6 +46,12 @@ can boot:
 
 1. Upon a successful build, you can find the uncompressed kernel image under
    `./vmlinux`.
+   
+   For aarch64, however, you're going to need `vmlinux.bin`. To get that, run
```

Comment:
Why do we need `vmlinux.bin` for aarch64 ?

Reply:
I just tried to run it on my NVIDIA Jetson Nano with `vmlinux` and I got this error:

https://github.com/firecracker-microvm/firecracker/blob/main/src/kernel/src/loader/mod.rs#L195

which happens because the `vmlinux` is an ELF file, but this loader does not expect an ELF file, it expects a .bin because it expects the ELF magic number to be at the 56th byte. When I ran it with `vmlinux.bin` it worked (but I got another error about KVM not opening, I guess my Jetson Nano kernel was not compiled with KVM support, still working on that)

Reply:
Thanks for the contribution. This is also linked to an issue we've opened recently: https://github.com/firecracker-microvm/firecracker/issues/2648
The way we've done the compilation on aarch64 in the past is by using `make Image`. The image is then placed
in `arch/arm64/boot/Image`

I suggest we update the doc with this information. I also think it's enough to state where the binary is placed, instead of issuing a copy command